tests/google_big_query/test_google_big_query.py

```diff
@@ -157,3 +159,152 @@ def test_retrieve_data(execute, connect, credentials, _fixture_credentials):
+    mocker.patch(
+        'toucan_connectors.google_big_query.google_big_query_connector.GoogleBigQueryConnector._connect',
+        return_value=Client,
+    )
+
+    mocker.patch(
+        'toucan_connectors.google_big_query.google_big_query_connector.GoogleBigQueryConnector._get_google_credentials',
+        return_value=Credentials,
+    )
+    connector = GoogleBigQueryConnector(
+        name='MyGBQ',
+        credentials=_fixture_credentials,
+        scopes=[
+            'https://www.googleapis.com/auth/bigquery',
+            'https://www.googleapis.com/auth/drive',
+        ],
+    )
+    assert connector.get_model() == [
+        {
+            'name': 'blabla',
+            'schema': 'baarrrr',
+            'database': 'myproject',
+            'type': 'view',
+            'columns': [
+                {'name': 'gogo', 'type': 'str'},
+                {'name': 'gaga', 'type': 'int64'},
+                {'name': 'gg', 'type': 'str'},
+            ],
+        },
+        {
+            'name': 'coucou',
+            'schema': 'foooo',
+            'database': 'myproject',
+            'type': 'table',
+            'columns': [
+                {'name': 'pingu', 'type': 'str'},
+                {'name': 'toto', 'type': 'int64'},
+                {'name': 'tante', 'type': 'str'},
+            ],
+        },
+        {
+            'name': 'tortuga',
+            'schema': 'taar',
+            'database': 'myproject',
+            'type': 'view',
+            'columns': [
+                {'name': 'hammer', 'type': 'str'},
+                {'name': 'to', 'type': 'int64'},
+                {'name': 'fall', 'type': 'str'},
+            ],
+        },
+    ]
+    assert (
+        mocked_query.call_args_list[0][0][0]
+        == "select C.table_name as name, C.table_schema as schema, T.table_catalog as database,\n                T.table_type as type,  C.column_name, C.data_type from foooo.INFORMATION_SCHEMA.COLUMNS C\n                JOIN foooo.INFORMATION_SCHEMA.TABLES T on C.table_name = T.table_name\n                where IS_SYSTEM_DEFINED='NO' AND IS_PARTITIONING_COLUMN='NO' AND IS_HIDDEN='NO'\n            \nUNION ALL\nselect C.table_name as name, C.table_schema as schema, T.table_catalog as database,\n                T.table_type as type,  C.column_name, C.data_type from baarrrr.INFORMATION_SCHEMA.COLUMNS C\n                JOIN baarrrr.INFORMATION_SCHEMA.TABLES T on C.table_name = T.table_name\n                where IS_SYSTEM_DEFINED='NO' AND IS_PARTITIONING_COLUMN='NO' AND IS_HIDDEN='NO'\n            \nUNION ALL\nselect C.table_name as name, C.table_schema as schema, T.table_catalog as database,\n                T.table_type as type,  C.column_name, C.data_type from taar.INFORMATION_SCHEMA.COLUMNS C\n                JOIN taar.INFORMATION_SCHEMA.TABLES T on C.table_name = T.table_name\n                where IS_SYSTEM_DEFINED='NO' AND IS_PARTITIONING_COLUMN='NO' AND IS_HIDDEN='NO'\n            "
```

Comment:
It's not a block point but for a good readability of the source code, i think it's better to have a multiline SQL query... and by using """ instead of ", what do you think ?

Reply:
okay